interfaces/kmod/kmod.go

```diff
@@ -34,11 +34,16 @@ func LoadModule(module string) error {
 }
 
 // loadModules loads given list of modules via modprobe.
-// Any error from modprobe interrupts loading of subsequent modules and returns the error.
+// Since different kernels may not have the requested module, we treat any
+// error from modprobe as non-fatal and subsequent module loads are attempted
+// (otherwise failure to load a module means failure to connect the interface
+// and the other security backends)
 func loadModules(modules []string) error {
 	for _, mod := range modules {
 		if err := LoadModule(mod); err != nil {
-			return err
```

Comment:
While not a semantic change we could just do this:

```
for _, mod := range modules {
    LoadModule(mod)  // ignore errors, they are logged internally by LoadModule
}
```

Reply:
Done